FFXIV_TexTools_CLI/Arguments.cs

```diff
@@ -79,6 +81,10 @@ public void ArgumentHandler(string[] args)
                         case "custom":
                             customImport = true;
                             continue;
+                        case "np":
```

Comment:
WOuld maybe -npc make more sense?

Also, what should ./exe pc -npc do? 
"Asked to run a problem check without a problem check. Exiting."?

Reply:
Ah yes, -npc would indeed make more sense. Running pc -npc will continue the problem check, -npc is only tied to the modpack import, as is also written in the argument's description.

Reply:
I guess that's fine for now. Might need to revisit if that flag becomes useful later